composer.json

```diff
@@ -22,10 +22,11 @@
     "ext-openssl": "*",
     "google/apiclient": "^2.12.2",
     "google/apiclient-services": "0.277.*",
-    "guzzlehttp/guzzle": "^5.3.3",
+    "guzzlehttp/guzzle": "^6.5.8",
     "true/punycode": "^2.0 <2.1.1"
   },
   "replace": {
+    "paragonie/random_compat": ">=2",
```

Comment:
`random_compat` is bundled in WordPress so this avoids loading via Composer, potentially loading an older version.

The version here is significant as any version below version 2 cannot be installed due to `roave/security-advisories`, so the constraint must only target versions which could actually be installed. This configuration causes the package _not_ to be installed.